disttask/framework/dispatcher/dispatcher_manager.go

```diff
@@ -58,6 +63,11 @@ func (dm *Manager) isRunningTask(taskID int64) bool {
 func (dm *Manager) delRunningTask(taskID int64) {
 	dm.runningTasks.Lock()
 	defer dm.runningTasks.Unlock()
+	task, err := dm.taskMgr.GetGlobalTaskByID(taskID)
```

Comment:
Why get task？

Reply:
For metrics label "task_type"、 "task_id"

Reply:
you can input `*proto.Task` here